test/tictactoe_ui_test.exs

```diff
@@ -3,26 +3,50 @@ defmodule TicTacToe.UI.Test do
   import ExUnit.CaptureIO
 
   test "can print the board" do
-    assert capture_io(fn -> TicTacToe.UI.start_game() end) == "123\n456\n789\n"
+    assert capture_io(fn -> TicTacToe.UI.start_game() end) == "\e[H\e[2J\n 1 | 2 | 3 \n===========\n 4 | 5 | 6 \n===========\n 7 | 8 | 9 \n"
   end
   
   test "can print the board after player one input" do
-    assert capture_io([input: "1"],fn ->  TicTacToe.UI.get_move({:x,[["1","2","3"],["4","5","6"],["7","8","9"]], :continue})end) == "Please enter a tile selection:X23\n456\n789\n"  
+    assert capture_io([input: "1"],fn ->  TicTacToe.UI.get_move({:x,[["1","2","3"],["4","5","6"],["7","8","9"]], :continue})end) == "\e[H\e[2J\nPlease enter a tile selection: \e[H\e[2J\n X | 2 | 3 \n===========\n 4 | 5 | 6 \n===========\n 7 | 8 | 9 \n"  
   end
 
   test "can print the board after player two input" do
-    assert capture_io([input: "2"],fn ->  TicTacToe.UI.get_move({:o,[["X","2","3"],["4","5","6"],["7","8","9"]], :continue})end) == "Please enter a tile selection:XO3\n456\n789\n"  
+    assert capture_io([input: "2"],fn ->  TicTacToe.UI.get_move({:o,[["X","2","3"],["4","5","6"],["7","8","9"]], :continue})end) == "\e[H\e[2J\nPlease enter a tile selection: \e[H\e[2J\n X | O | 3 \n===========\n 4 | 5 | 6 \n===========\n 7 | 8 | 9 \n"  
   end
 
   test "prints correct message after duplicate tile selected" do
```

Comment:
Again, the test names aren't quite consistent here. Sometimes you say `can do something` and sometimes you say `does something`